pkg/operator/target_config_reconciler.go

```diff
@@ -37,8 +36,11 @@ import (
 	"github.com/openshift/jobset-operator/pkg/operator/operatorclient"
 )
 
-const WEBHOOK_CERTIFICATE_NAME = "webhook-server-cert"
-const CERT_MANAGER_INJECT_CA_ANNOATION = "cert-manager.io/inject-ca-from"
+const (
+	WebhookCertificateName        = "webhook-server-cert"
+	WebhookCertificateSecretName  = "jobset-serving-cert"
```

Comment:
this names should be interchanged. `jobset-serving-cert` is Certificate name. `webhook-server-cert` is secret name. Usages are correct though -- secretlister uses `WebhookCertificateName` which correctly points to `webhook-server-cert`

Reply:
But this can be applied in a followup PRs, because there is no functional issue.